client/app/components/app-header/app-header.html

```diff
@@ -124,6 +124,19 @@
             <li>
               <a ng-click="$ctrl.logout()">Log out</a>
             </li>
+
+            <li ng-if="$ctrl.currentUser.hasPermission('super_admin')" class="divider"></li>
+            <li ng-if="$ctrl.currentUser.hasPermission('super_admin')" class="dropdown-menu__version">
```

Comment:
Why not show version to everyone?

Reply:
Just to avoid visual clutter.
Maybe just hide the upgrade button from non-admins?

Reply:
We should definitely show the version to users.

About the upgrade button: while non admins have no use for it (they can't upgrade), they can nag the admin to upgrade :) But if you feel it's a distraction, we can hide it for non admins.

Reply:
If it's green - it's a distraction.
If it's not green - it's purpose isn't clear (I experimented with many colors).
So, I'm for hiding it from non-admins.

Reply:
Done